asv_bench/benchmarks/stat_ops.py

```diff
@@ -102,7 +102,7 @@ def time_average_old(self, constructor, pct):
 class Correlation:
 
     params = [["spearman", "kendall", "pearson"], [True, False]]
-    param_names = ["method", "use_bottleneck"]
+    param_names = ["method"]
 
     def setup(self, method, use_bottleneck):
```

Comment:
Can you remove the parametrization of `use_bottleneck` here and in other methods? Also can remove the import / USE_BOTTLENECK assignment in setup

Reply:
Done!